be/db/dev-db/README.md

```diff
@@ -8,6 +8,18 @@ as:
 
 [dbeaver github](https://github.com/dbeaver/dbeaver)
 
+## Table of contents
+
+- [Accessing the database](#accessing-the-database)
+
+- [Running the development database in docker](#running-the-development-database-in-docker)
+
+- [Stopping the development database](#stopping-the-development-database)
+
+- [Adding example data and tables](#adding-example-data-and-tables)
+
+- [Troubleshooting](#troubleshooting)
+
 ## Accessing the database
 
 Access the database using adminer at: 'localhost:8080'. Make sure to select
```

Comment:
maybe add proper link so it would be clickable.

Reply:
It is clickable. try here -> https://github.com/TPT-Loane/TPT-Loane/tree/db-documentation/be/db/dev-db

Reply:
'localhost:8080' is not clickable! it doesn't even have proper protocol.

Reply:
aaa, I thought you meant the toc links.
In this case, might as well add some parameters to the url to automatically fill in the user and db parts.
`http://localhost:8080/?server=db&username=postgres&db=postgres`

Reply:
@OskarSchamardin fix plz